nhsd-backend/README.md

```diff
@@ -0,0 +1,11 @@
+# NHSD Backend
+
+This is the configuration for the NSHD DAE hosted backend.
```

Comment:
DAE?

Reply:
Data Access Environment.

The intended audience here is for NHSD folks, but it worth clarifying.